templates/xfs_quota.epp

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# Script from https://git.ncsa.illinois.edu/ici-monitoring/ici-developed-checks/-/blob/main/misc/xfs_quota.sh?ref_type=heads
+
+tfile=$(mktemp /tmp/xfs.XXXXXXX)
+path=<%= $profile_xfs_quota::quota_path %>
```

Comment:
Again somewhere in here is where you would ideally want to support more than one `quota_path`.

While it is somewhat clunky, you can embed puppet code logic loops in EPP templates. I can point you to some examples if that is useful.

Reply:
Sounds good. If it's ok I will leave as is for now, with the intent to expand functionality once the script supports multiple paths (issue created in ici-monitoring control repo)